sdk-api-src/content/cfapi/nf-cfapi-cfcreateplaceholders.md

```diff
@@ -58,6 +58,8 @@ Creates one or more new placeholder files or directories under a sync root tree.
 
 Local directory path under which placeholders are created.
```

Comment:
The real problem is the use of the word "under", which implies that this can be a grandparent or higher ancestor of the final placeholder.

I would remove the added paragraph (since it's covered by the **CF_PLACEHOLDER_CREATE_INFO** below) and change this sentence to

> Path to the local directory in which the placeholders are created. This path must be under the sync root of the provider.

Reply:
I agree rewording and removing the duplicate paragraph is an improvement.

Reply:
Great. I will make the suggested changes and send another PR.